packages/vhd-lib/src/Vhd/VhdAbstract.integ.spec.js

```diff
@@ -139,3 +150,81 @@ test('It create , rename and unlink alias', async () => {
     expect(await fs.exists(aliasFileNameRenamed)).toEqual(false)
   })
 })
+
+test('it can create a vhd stream', async () => {
+  const initalSize = 6
+  const rawFileName = `${tempDir}/randomfile`
+  await createRandomFile(rawFileName, initalSize)
+  const vhdFileName = `${tempDir}/vhd.vhd`
+  await convertFromRawToVhd(rawFileName, vhdFileName)
+  const bat = Buffer.alloc(512)
+
+  await Disposable.use(async function* () {
+    const handler = yield getSyncedHandler({ url: 'file://' + tempDir })
+
+    // mark first block as unused
+    await handler.read('vhd.vhd', bat, 1024 + 512)
```

Comment:
I'm afraid that this might break in the future because we have no guarantee of where the BAT really is.